packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -373,6 +373,10 @@ class DefaultTabController extends StatefulWidget {
   /// match [TabBar.tabs]'s and [TabBarView.children]'s length.
   ///
   /// The [initialIndex] argument must not be null.
+  ///
+  /// See also:
+  ///
+  ///  * [PageStorageKey] are used to remember the index.
```

Comment:
This "see also" is strange. It should probably just be a paragraph of the docs that PageStorageKeys are supported and what they can be used for.